### PR TITLE
Multiple swipers on the same view

### PR DESCRIPTION
**Pull request by vongohren**
Its not possible now to have multiple swipers in the same container
I cant re render the component and expecting the content to change.
We have a issue where we have a slider that is dependant on an dropdown value.
So when this value changes, the slider content changes.

But since the component is already mounted, there is not an entrance to newing up an new Swiper.

This is my suggetstion, works for us.

### CODE REVIEW

File: react-swipe.js [hunk 1]
```diff
@@ -47,7 +47,8 @@
       disableScroll   : React.PropTypes.bool,
       stopPropagation : React.PropTypes.bool,
       callback        : React.PropTypes.func,
-      transitionEnd   : React.PropTypes.func
+      transitionEnd   : React.PropTypes.func,
+      newSwiper       : React.PropTypes.bool
```
Comment (voronianski) [inline react-swipe.js]: @Snorlock I think should be a better name for this. Maybe `reinitSwipeOnUpdate`?

File: react-swipe.js [hunk 2]
```diff
@@ -55,6 +56,9 @@
     },
 
     componentDidUpdate: function () {
+      if(this.props.newSwiper) {
```
Comment (voronianski) [inline react-swipe.js]: @Snorlock  `if(` - space is missing.

File: react-swipe.js (hunk 1 above)
Comment (vongohren) [inline react-swipe.js]: @voronianski  done

File: react-swipe.js (hunk 2 above)
Comment (vongohren) [inline react-swipe.js]: done

**Comment by voronianski** [comment]
@Snorlock please rebase to be able to merge.

**Comment by voronianski** [comment]
@Snorlock I'm not able to merge this because `This branch has conflicts that must be resolved` could you rebase with latest master?

**Comment by vongohren** [comment]
yeah il do it now. Its just that I used the online editor.

**Comment by vongohren** [comment]
Done

**Comment by voronianski** [comment]
@Snorlock :+1:

**Pull request merged**